### ISSUE TITLE
Error in if (lev[1] != inter) { : the condition has length > 1

### ISSUE THREAD

**Issue by AvniMann14** [jiabowang/GAPIT#162]
i am facing this error while numericalization of hapmap file

**Comment by AvniMann14** [comment]
myGAPIT <- GAPIT(
+   Y=myY,
+   G=myG,
+   model="GLM",
+   PCA.total=3
+ )
[1] "--------------------- Welcome to GAPIT ----------------------------"
[1] "GLM"
[1] "--------------------Processing traits----------------------------------"
[1] "Phenotype provided!"
[1] "The 1 model in all."
[1] "GLM"
[1] "GAPIT.DP in process..."
[1] "Converting genotype..."
[1] "Converting HapMap format to numerical under model of Middle"
[1] "Perform numericalization"
Error in if (lev[1] != inter) { : the condition has length > 1

**Comment by jiabowang** [comment]
Please check whether there are more than 3 genotype in someone marker.

**Comment by jiabowang** [comment]
@rachelnaegele could you share your data with me? So I can help figure out the reason.

**Comment by rachelnaegele** [comment]
@jiabowang Of course! Here is the hapmap (generated in Tassel) file. I ended up getting GAPIT to work by converting it to a numerical genotype in Tassel and then using that plus the position file. I never got the hapmap file to work. I used the "Get Position List" to see what the base calls were for each marker, and filtered  sites with more than 3 bases listed.

[strxA6161_sorted_3geno.hmp.txt](https://github.com/user-attachments/files/19705820/strxA6161_sorted_3geno.hmp.txt)

**Comment by jiabowang** [comment]
@rachelnaegele I noticed that there are many multiple variates type in your HapMap file.
Such as 
rh_chr1_741516	A/G/C	1	741413
rh_chr1_1170229	A/G/C	1	1170119	
Those make this error.
Please remove them or re-code them.
